scripts/sysman/EXT_Exp_VirtualFunctionManagement.rst

```diff
@@ -0,0 +1,43 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_virtual_function_management:
+
+========================================
+ Virtual Function Management Extension
+========================================
+
+API
+----
+* Functions
+
+    * ${s}EnumActiveVirtualFunctionExp
+    * ${s}GetVFPropertiesExp
+    * ${s}GetVFMemoryUtilizationExp
```

Comment:
If we use camel case here we would be having Vf instead of VF right ?

Reply:
Since it's commonly abbreviated with 2 caps, I think this is OK.